src/main/java/com/twentyonec/ItemsLogger/utils/Storage.java

```diff
@@ -255,4 +251,10 @@ public ItemPlayer retrieveItemPlayer(UUID uuid, String date, String time) {
 			return null;
 		}
 	}
+
+	public void deleteLogs(final int days) {
+		plugin.debugMessage("Attempting to delete logs older than " + days + " days.");
+		final String sql = "DELETE FROM itemslogger WHERE date < DATE_SUB(CURDATE(), INTERVAL ? DAY)";
```

Comment:
You might prefer using `DATEDIFF(CURDATE(), date) > ?` instead. It returns the exact difference in days between the two provided values, instead of performing arithmetic and then comparing dates. Much easier and cleaner.
```suggestion
		final String sql = "DELETE FROM itemslogger WHERE DATEDIFF(CURDATE(), date) > ?;";
```